### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by ccleva**
Subtree split from the parquet submodule in https://github.com/ccleva/tablesaw/tree/feature/parquet-read-write

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>tech.tablesaw</groupId>
+		<artifactId>tablesaw-parent</artifactId>
+		<version>0.38.3-SNAPSHOT</version>
```
Comment (dbalazuc) [inline pom.xml:10]: Tablesaw dependencies use 0.38.3.SNAPSHOT. They aren't available. Use 0.38.2 instead.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
+
+	<dependencies>
+		<dependency>
+			<groupId>tech.tablesaw</groupId>
+			<artifactId>tablesaw-core</artifactId>
+			<version>0.38.3-SNAPSHOT</version>
```
Comment (dbalazuc) [inline pom.xml:116]: SNAPSHOT

**Review by dbalazuc — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (ccleva) [reply pom.xml:10]: Yeahs this comes from the fork, we need to find a way to support all released versions (starting with 0.38.2) while working with the snapshot to be able to release the new version at the same time. Suggestions are welcome.

File: pom.xml (hunk 2 above)
Comment (ccleva) [reply pom.xml:116]: See above

**Review by dbalazuc — APPROVED** [review]

**Comment by dbalazuc** [comment]
.gitignore references files that don't exist. It can probably be reduced to 
```
.mvn/
target/
```

**Pull request merged**